lib/model/store.dart

```diff
@@ -908,15 +911,31 @@ class UpdateMachine {
     store.updateMachine = this;
   }
 
-  /// Load the user's data from the server, and start an event queue going.
+  /// Load data for the given account ID from the server,
+  /// and start an event queue going.
+  ///
+  /// The account for `accountId` must exist.
+  ///
+  /// Throws [AccountNotFoundException] after an async gap
+  /// if the account has been removed.
```

Comment:
```suggestion
  /// Throws [AccountNotFoundException] if after any async gap
  /// the account has been removed.
```

Otherwise it sounds like it's saying: if the account has been removed, throws said exception, but first causes an async gap.